static/app/components/events/autofix/autofixSolution.tsx

```diff
@@ -418,10 +431,12 @@ export function AutofixSolution(props: AutofixSolutionProps) {
     );
   }
 
+  const changesDisabled = !props.repos.every(repo => repo.is_readable);
+
```

Comment:
should this be `every` or `some`? Do we want to allow coding if at least one repo is readable?

Reply:
good point, it should be `props.repos.every(repo => !repo.is_readable);` actually

Reply:
but `!props.repos.some(repo => repo.is_readable);` is more efficient